smartcontractkit/external-adapters-js
# Fix API 422 response for free plan

## Description

Discussed on slack with Austin.
Nomics requires per-page param for free tier. 

## Changes

- Added required per-page param to nomics crypto endpoint. Others are not available on the free tier.


## Steps to Test

1. Nomics free plan API key
2. Call the crypto endpoint
3. Will receiver 422 Response: Your plan requires that you provide a per-page argument. See http://docs.nomics.com/ for more information.

## Quality Assurance
